### PR TITLE
Limelight programming

### PR DESCRIPTION
**Pull request by lanc33llis**
limelight, turret, servo/linear actuator basics

### CODE REVIEW

File: .metals/metals.lock.db [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#FileLock
```
Comment (legoguy1000) [inline .metals/metals.lock.db]: What is this file?

Comment (legoguy1000) [reply .metals/metals.lock.db]: and the metals.h2.db?

**Review by legoguy1000 — COMMENTED** [review]
Seems good overall.  There are a couple of files i don't recognize.  Will have the minions take a peak tomorrow and then we'll merge.

File: .metals/metals.lock.db (hunk 1 above)
Comment (lanc33llis) [reply .metals/metals.lock.db]: scala files generated automatically by an extension, doesn't matter delete or leave

**Review by klee111287 — APPROVED** [review]

**Pull request merged**